.github/workflows/ci-pipeline.yml

```diff
@@ -1,10 +1,11 @@
 name: CI Automate
 on:
   push:
-    branches:
-
+    branches: [ft-manage-ratings]
```

Comment:
Please avoid editing this file since this mean that the action will run only on this branch, like really this branch

Restore this line